aeron-cluster/src/main/java/io/aeron/cluster/Election.java

```diff
@@ -59,6 +59,7 @@
     private long timeOfLastStateChangeNs;
     private long timeOfLastUpdateNs;
     private long timeOfLastCommitPositionUpdateNs;
+    private long timeOfLastLeaderMessageReceivedNs;
```

Comment:
Do we need to think about initialising this value when we start and reset an election?  What would be a sensible default when the election starts?  Should it be 0 as we are assuming that the leadership timeout has already expired to trigger the election or should it be initialised to the timestamp of when the election starts.

If the election resets, should we re-initialise this field?

Reply:
This can be solved if we move the field to the `ConsensusModuleAgent` and have it track the time instead. Then each election will automatically have access to the right value.

Reply:
The time is tracked by the `ConsensusModuleAgent` now.
I don't think we need to do anything special if election resets.

Reply:
I think you're correct.  If the election is reset, the last time that a message has been received from the leader doesn't change.